about.php

```diff
@@ -0,0 +1,48 @@
+<?php
+require 'dbConn.php';
+function retrieve_data($db)
+{
+    $retrieve_statement = $db->prepare('SELECT `content`, `picture_link`, `phone_number`, `email` FROM `about_me` WHERE `id` = 1;');
```

Comment:
You do not need to prepare this. Could just use $retrieve_statement = $db->query as you are not using any user submitted data :-)